src/js/dialogs-manager.js

```diff
@@ -330,26 +336,31 @@
 				classes: {
 					globalPrefix: parentSettings.classPrefix,
 					prefix: parentSettings.classPrefix + '-' + widgetName,
-					preventScroll: parentSettings.classPrefix + '-prevent-scroll'
+					preventScroll: parentSettings.classPrefix + '-prevent-scroll',
 				},
 				selectors: {
-					preventClose: '.' + parentSettings.classPrefix + '-prevent-close'
+					preventClose: '.' + parentSettings.classPrefix + '-prevent-close',
 				},
 				container: 'body',
 				preventScroll: false,
 				iframe: null,
 				closeButton: false,
 				closeButtonOptions: {
 					iconClass: parentSettings.classPrefix + '-close-button-icon',
-					attributes: {},
+					attributes: {
+						role: 'button',
+						'tabindex': 0,
+						'aria-label': 'Close',
+						href: 'javascript:void(0);',
```

Comment:
```suggestion
```
Is it necessary?

Reply:
Yes, after we change the element to `a` it must be valued.